main.go

```diff
@@ -184,9 +306,20 @@ func main() {
 	go func(results <-chan *zeroconf.ServiceEntry) {
 		fmt.Printf("[scanner] looking for Shelly devices using mDNS (%ds timeout)...\n", int(scanTimeout.Seconds()))
 		for entry := range results {
+			entry := entry
```

Comment:
```suggestion
```

Reply:
I think we need this until Go 1.22 (https://go.dev/blog/loopvar-preview) or enable `GOEXPERIMENT=loopvar`